docs/en/ingest-management/tab-widgets/install-layout.asciidoc

```diff
@@ -7,10 +7,8 @@
 Main {agent} configuration
 `/Library/Elastic/Agent/fleet.enc`::
 Main {agent} {fleet} encrypted configuration
-`/Library/Elastic/Agent/data/elastic-agent-*/logs/elastic-agent-json.log`::
-Log files for {agent}footnote:lognumbering[Logs file names end with a date and optional number: log-date.ndjson, log-date-1.ndjson, and so on as new files are created during rotation.]
-`/Library/Elastic/Agent/data/elastic-agent-*/logs/default/*-json.log`::
-Log files for {beats} shippersfootnote:lognumbering[]
+`/Library/Elastic/Agent/data/elastic-agent-*/logs/elastic-agent.ndjson`::
+Log files for {agent} and beats shippers footnote:lognumbering[Logs file names end with a date (`YYYYMMDD`) and optional number: `elastic-agent-YYYYMMDD.ndjson`, `elastic-agent-YYYYMMDD-1.ndjson`, and so on as new files are created during rotation.]
```

Comment:
```suggestion
Log files for {agent} and {beats} shippers footnote:lognumbering[Logs file names end with a date (`YYYYMMDD`) and optional number: `elastic-agent-YYYYMMDD.ndjson`, `elastic-agent-YYYYMMDD-1.ndjson`, and so on as new files are created during rotation.]
```

Just adding the attribute for "Beats" that's defined over [here](https://github.com/elastic/docs/blob/7470e3516e5298e2c540bd22f3281314c7e863e2/shared/attributes.asciidoc#L287).